src/transformers/modeling_utils.py

```diff
@@ -808,7 +809,13 @@ def _load_state_dict_into_meta_model(
     for old_key, new_key in zip(old_keys, new_keys):
         state_dict[new_key] = state_dict.pop(old_key)
 
-    for param_name, param in state_dict.items():
+    # Show shard-level progress. Useful to monitor quantization progress
+    quant_show_progress = False
+    if hf_quantizer is not None:
+        if hasattr(hf_quantizer, "show_progress"):
+            quant_show_progress = hf_quantizer.show_progress
```

Comment:
Although - sticking with above we should just remove entirely 

```suggestion
    if hf_quantizer is not None and hasattr(hf_quantizer, "show_progress"):
        quant_show_progress = hf_quantizer.show_progress
```

Reply:
Alright, I removed the quant progress as requested: https://github.com/huggingface/transformers/pull/29637/commits/cc14c2118ed359e600d7c6758200c840284a1056